### ISSUE TITLE
First two characters of file names truncated on windows

### ISSUE THREAD

**Issue by foxbunny** [gpbl/webpack-cleanup-plugin#24]
I'm using this plugin on windows, and it appears that it likes to truncate the first two characters of the file name (rest of the path is OK). For example, I have a file

```
.\public\admin-1f24c22b69306f2aae0f.js
```

When I turn on preview flag, this file is printed:

```
./public/min-1f24c22b69306f2aae0f.js
```

Full traceback:

```
Error: ENOENT: no such file or directory, unlink 'C:\project\public\min-1f24c22b69306f2aae0f.js'
    at fs.unlinkSync (fs.js:996:18)
    at Array.forEach (native)
    at Compiler.<anonymous> (C:\project\node_modules\webpack-cleanup-plugin\lib\WebpackCleanupPlugin.js:55:17)
    at Compiler.applyPlugins (C:\project\node_modules\tapable\lib\Tapable.js:26:37)
    at Compiler.<anonymous> (C:\project\node_modules\webpack\lib\Compiler.js:193:12)
    at Compiler.emitRecords (C:\project\node_modules\webpack\lib\Compiler.js:282:37)
    at Compiler.<anonymous> (C:\project\node_modules\webpack\lib\Compiler.js:187:11)
    at C:\project\node_modules\webpack\lib\Compiler.js:275:11
    at Compiler.applyPluginsAsync (C:\project\node_modules\tapable\lib\Tapable.js:60:69)
    at Compiler.afterEmit (C:\project\node_modules\webpack\lib\Compiler.js:272:8)
    at Compiler.<anonymous> (C:\project\node_modules\webpack\lib\Compiler.js:267:14)
    at C:\project\node_modules\async\lib\async.js:52:16
    at done (C:\project\node_modules\async\lib\async.js:246:17)
    at C:\project\node_modules\async\lib\async.js:44:16
    at C:\project\node_modules\graceful-fs\graceful-fs.js:43:10
    at FSReqWrap.oncomplete (fs.js:111:15)
```

Plugin version: 0.4.1
Webpack version: 1.14.0
Node: 7.3.0 (64-bit)

**Comment by foxbunny** [comment]
It appears that normalizing the `fromPath` parameter in `getFiles()` fixes this issue.

```javascript
function getFiles(fromPath) {
  fromPath = _path.normalize(fromPath)  // <--- here
  var exclude = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : [];
  // ....
}
```

**Comment by ghost** [comment]
Unfortunately for me, normalizing the path isn't sufficient.

In the function that creates a substring of `p`, I get the following values - 
`p` = `C:\Users\hbetts\Workspace\application\dist\application.js`
Substring of `p` = `pplication.js`

Removing the `+ 1` from the call to `substr` resolved the issue for me (in addition to the path normalization).

**Comment by gpbl** [comment]
This should be fixed in v0.4.2! Please try it and reopen if still not working as expected (I can't actually test on Windows right now).

Thanks everybody!
